src/client/src/components/chatpage.js

```diff
@@ -5,7 +5,7 @@ const ConnectStatus = require('./connectStatus')
 const Room = require('./room')
 const socket = require('../utils/sockets/ws-routing')
 const protocols = require('../utils/io-protocol')
-const HOMEPAGE_PATH = '/'
+const HOMEPAGE_PATH = '/', EMPTY = ''
```

Comment:
Could you avoid declarations with comma? It's better to state each variable in clear, and very visible way.

Reply:
Also, change it to `USERNAMES_PLACEHOLDER` maybe?